packages/sources/tiingo-test/src/ws-utils.ts

```diff
@@ -0,0 +1,25 @@
+import { TransportGenerics } from '@chainlink/external-adapter-framework/transports'
+import {
+  WebsocketReverseMappingTransport,
+  WebSocketTransport,
+} from '@chainlink/external-adapter-framework/transports/websocket'
+
+// Once WebsocketTransportGenerics type is exported from EA framework this can be removed
+type WebsocketTransportGenerics = TransportGenerics & {
```

Comment:
I think we can get a quick PR up for this so we don't have to come up to tweak just this later. We can maybe bundle it with https://github.com/smartcontractkit/ea-framework-js/pull/52

Reply:
I've updated my PR